src/content/RepoPage/RepoPage.js

```diff
@@ -1,7 +1,70 @@
 import React from 'react';
+import RepoTable from './RepoTable';
 
 const RepoPage = () => {
-  return <div>REPO PAGE</div>;
+  const headers = [
+    {
+      key: 'name',
+      header: 'Name',
+    },
+    {
+      key: 'createdAt',
+      header: 'Created',
+    },
+    {
+      key: 'updatedAt',
+      header: 'Updated',
+    },
+    {
+      key: 'issueCount',
+      header: 'Open Issues',
+    },
+    {
+      key: 'stars',
+      header: 'Stars',
+    },
+    {
+      key: 'links',
+      header: 'Links',
+    },
+  ];
+
+  const rows = [
+    {
+      id: '1',
+      name: 'Repo 1',
+      createdAt: 'Date',
+      updatedAt: 'Date',
+      issueCount: '123',
+      stars: '456',
+      links: 'Links',
+    },
+    {
+      id: '2',
+      name: 'Repo 2',
+      createdAt: 'Date',
+      updatedAt: 'Date',
+      issueCount: '123',
+      stars: '456',
+      links: 'Links',
+    },
+    {
+      id: '3',
+      name: 'Repo 3',
+      createdAt: 'Date',
+      updatedAt: 'Date',
+      issueCount: '123',
+      stars: '456',
+      links: 'Links',
+    },
+  ];
+  return (
+    <div className="bx--grid bx--grid--full-width bx--grid--no-gutter repo-page">
+      <div className="bx--row repo-page__r1">
+        <RepoTable headers={headers} rows={rows} />
```

Comment:
Missing `<div className="bx--col-lg-16">` wrapper

Reply:
@kellychurchill The issue should be fixed.